SalesforceCommerceCloud/pwa-kit
# [MRT-Reference-App] Add example.txt file for testing Brotli compression in /mobify/bundle/* path

In order to test Brotli compression for the bundle path we require a test file that is pushed to the cdn-origin bundles S3 bucket and is at least 1,000 bytes. More details on compression criteria [here](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/ServingCompressedFiles.html)

# Description

Add `static/example.txt ` file to `template-mrt-reference-app`

# Types of Changes


> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Add `static/example.txt ` file to `template-mrt-reference-app`

# How to Test-Drive This PR

1. Push the MRT reference app
  ```
  npm run push -- -m 'mrt-ref-app-with-example-txt' -s 'YOUR_PROJECT' --credentialsFile /Users/j.ha/.mobify--cloud-jinsu.mobify-staging.com --cloud-origin https:/cloud-jinsu.mobify-staging.com
  ```
2. PCSK into the MRT Non-Production Root account
3. Navigate to S3 and verify example.txt is in the CDN bucket -> `cloud-cdn-origin-YOUR_NAME-766791126171/sites/YOUR_PROJECT/bundles/BUNDLE_NUMBER/static/example.txt`
4. Verify the permission of example.txt has: Object Read access for Everyone

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
